frontend/src/pages/profile/AccountInformation.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(4, 4, 0, 4),
+  },
+  divider: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+  account: {
+    margin: theme.spacing(4),
+  },
+  label: {
+    height: '100%',
+    display: 'flex',
+    alignItems: 'center',
+  },
+  header: {
+    fontWeight: 'bold',
+  },
+}));
+
+const AccountInformation = (props) => {
+  const classes = useStyles();
+
+  // TODO: contains mocked data, need to retrieve this via store
+  return (
+    <Grid container>
+      <Grid item xs={12} md={12} lg={12} className={classes.divider}>
```

Comment:
als alles 12 is, hoef je alleen `xs={12}` te doen, om dat xs betekend *vanaf* extra small moet het 12 columns in beslag nemen (dus automatisch voor md, lg, xl ook)